action_cable_client.gemspec

```diff
@@ -23,12 +22,12 @@ Gem::Specification.new do |s|
 
   s.test_files = s.files.grep(%r{^(test|spec|features)/})
 
-  s.required_ruby_version = '>= 2.2.2'
+  s.required_ruby_version = '>= 2.3'
```

Comment:
Is there anything in particular that isn't supported in ruby 2.2.2 or higher?

Reply:
I changed this due to `rubocop` warning about miss-matching version between `TargetRubyVersion` and `s.required_ruby_version`. As 2.2 is not maintained anymore, I choose to target 2.3.